qiskit/circuit/quantumcircuit.py

```diff
@@ -4146,7 +4457,45 @@ def from_qasm_str(qasm_str: str) -> "QuantumCircuit":
 
     @property
     def global_phase(self) -> ParameterValueType:
-        """The global phase of the current circuit scope in radians."""
+        """The global phase of the current circuit scope in radians.
+
+        Example:
+
+            .. plot::
+                :include-source:
+                :nofigs:
+                :context: reset
+
+                from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+
+                qreg_q = QuantumRegister(2, 'q')
+                creg_c = ClassicalRegister(2, 'c')
+                circuit = QuantumCircuit(qreg_q, creg_c)
+
+                circuit.h(qreg_q[0])
+                circuit.cx(qreg_q[0], qreg_q[1])
+
```

Comment:
What do you think about removing explicit registers here?

```suggestion
                from qiskit import QuantumCircuit

                circuit = QuantumCircuit(2)
                circuit.h(0)
                circuit.cx(0, 1)
```